docs.md

```diff
@@ -0,0 +1,85 @@
+| `tags`                    | `false` (except on tag event)       | Fetches tags when set to true, default is false if event is not tag else true                                                          |
+| `submodule_overrides`     | _none_                              | Override submodule urls                                                                                                                |
+| `submodule_update_remote` | `false`                             | Pass the --remote flag to git submodule update                                                                                         |
+| `custom_ssl_path`         | _none_                              | Set path to custom cert                                                                                                                |
+| `custom_ssl_url`          | _none_                              | Set url to custom cert                                                                                                                 |
+| `backoff`                 | `5sec`                              | Change backoff duration                                                                                                                |
+| `attempts`                | `5`                                 | Change backoff attempts                                                                                                                |
+| `branch`                  | $CI_COMMIT_BRANCH                   | Change branch name to checkout to                                                                                                      |
+| `partial`                 | `true` (except if tags are fetched) | Only fetch the one commit and it's blob objects to resolve all files, overwrite depth with 1                                           |
+| `home`                    |                                     | Change HOME var for commands executed, fail if it does not exist                                                                       |
+| `remote`                  | $CI_REPO_CLONE_URL                  | Set the git remote url                                                                                                                 |
+| `sha`                     | $CI_COMMIT_SHA                      | git commit hash to retrieve (use `sha: ''` to clone the `ref`)                                                                         |
+| `ref`                     | $CI_COMMIT_REF                      | Set the git reference to retrieve (use `ref: refs/head/a_branch` and `sha: ''` to retrieve the head commit from the "a_branch" branch) |
+| `path`                    | $CI_WORKSPACE                       | Set destination path to clone to                                                                                                       |
```

Comment:
Avaliable options are:
- PLUGIN_RELEASE_BRANCH default is CI_REPO_DEFAULT_BRANCH
- PLUGIN_GITHUB_TOKEN,
- PLUGIN_GIT_EMAIL,